code/modules/mob/living/simple_animal/abnormality/waw/silence.dm

```diff
@@ -0,0 +1,66 @@
+//Coded by me, Kirie Saito!
+/mob/living/simple_animal/hostile/abnormality/silence
+	name = "The Price of Silence"
+	desc = "A scythe with a clock attached, quitely ticking."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "silence"
+	threat_level = WAW_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(0, 0, 30, 40, 40),
+		ABNORMALITY_WORK_INSIGHT = 0,
+		ABNORMALITY_WORK_ATTACHMENT = list(0, 0, 10, 20, 30),
+		ABNORMALITY_WORK_REPRESSION = list(0, 0, 50, 45, 45)
+			)
+	start_qliphoth = 1
+	work_damage_amount = 10
```

Comment:
This is the same as every other pale damage working abno we have, I'm pretty sure. At least it's the same as Rose and WN I think. can we reduce this to like, 8?

Reply:
I looked at JBird and it was the same, Rose and WN are 14 pale

Reply:
Ah, fair then.